challenges/2021_cloud_base_height/01_send_data_to_zarr.ipynb

```diff
@@ -0,0 +1,4235 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.        dev_indiv_files = os.environ["SCRATCH"] + "/cbh_data/dev_indiv/" + "*.pp"
</pre><p>I see you've imported pathlib, it would be good to describe your paths as pathlib objects, as good python 3 practice.</p><p>it is a bit of a pain that iris doesn't understand pathlib.Path objects, but that will change at is otherwise still useful I think</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MetOffice/data_science_cop/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>